src/openzaak/components/documenten/tests/test_filters.py

```diff
@@ -1,18 +1,379 @@
+            data = response.json()
+            self.assertEqual(data["count"], 2)
+            self.assertEqual(data["results"][0]["auteur"], "Bernard")
+            self.assertEqual(data["results"][1]["auteur"], "Bert")
+
+    def test_beschrijving(self):
+        EnkelvoudigInformatieObjectFactory.create(
+            beschrijving="Random Item that won't be retrieved"
+        )
+        EnkelvoudigInformatieObjectFactory.create(beschrijving="Lorem Ipsum")
+        EnkelvoudigInformatieObjectFactory.create(
+            beschrijving="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+        )
+
+        with self.subTest("exact"):
+            response = self.client.get(
+                reverse(EnkelvoudigInformatieObject),
+                {
+                    "beschrijving": "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+                },
+            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            data = response.json()
+            self.assertEqual(data["count"], 1)
+            self.assertEqual(
+                data["results"][0]["beschrijving"],
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+            )
+
+        with self.subTest("contains"):
+            response = self.client.get(
+                reverse(EnkelvoudigInformatieObject), {"beschrijving": "Lorem ipsum"},
+            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            data = response.json()
+            self.assertEqual(data["count"], 2)
+            self.assertEqual(data["results"][0]["beschrijving"], "Lorem Ipsum")
+            self.assertEqual(
+                data["results"][1]["beschrijving"],
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+            )
+
+    def test_titel(self):
+        EnkelvoudigInformatieObjectFactory.create(titel="A...")
+        EnkelvoudigInformatieObjectFactory.create(titel="Lorem")
+        EnkelvoudigInformatieObjectFactory.create(titel="Lorem Ipsum")
+
+        with self.subTest("exact"):
+            response = self.client.get(
+                reverse(EnkelvoudigInformatieObject), {"titel": "Lorem Ipsum"},
+            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            data = response.json()
+            self.assertEqual(data["count"], 1)
+            self.assertEqual(data["results"][0]["titel"], "Lorem Ipsum")
+
+        with self.subTest("contains"):
+            response = self.client.get(
+                reverse(EnkelvoudigInformatieObject), {"titel": "Lorem"},
+            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            data = response.json()
+            self.assertEqual(data["count"], 2)
+            self.assertEqual(data["results"][0]["titel"], "Lorem")
+            self.assertEqual(data["results"][1]["titel"], "Lorem Ipsum")
+
+    def test_trefwoorden(self):
```

Comment:
I appreciate test coverage, but we already have tests for trefwoorden in [test_enkelvoudiginformatieobject.py](https://github.com/open-zaak/open-zaak/pull/1615/files?diff=unified&w=0#diff-f0d7f7a1ab4199b773388ce72a67d1e869d68e2f450658b21114b09d7dc8ad66)

If you want to refactor test you can move all filter tests from `test_enkelvoudiginformatieobject.py` to `test_filters (or vice versa)